### ISSUE TITLE
regression testing: traces

### ISSUE THREAD

**Issue by inolen** [inolen/redream#222]
We need to start building up a database of small traces (maybe I could add a button to capture a single-frame trace) such that we have a database moving forward of problematic areas in each game rendering-wise.

Additionally, the retrace tool should be extended to dump out pngs from these traces, so we can compare them for regressions on each commit.

**Comment by literalmente-game** [comment]
Traces in general are kind of big to upload here (because of the file size limit of Github), would thee files need to be uploaded to external sites like before or you're thinking about another solution?

**Comment by inolen** [comment]
Right, the size is why I think having a mode to capture a single frame should work out well. I'll probably create a separate repo to store these in.
